app/src/main/kotlin/me/amrbashir/hijriwidget/preference_activity/LauncherIcon.kt

```diff
@@ -3,33 +3,41 @@ package me.amrbashir.hijriwidget.preference_activity
 import android.content.ComponentName
 import android.content.Context
 import android.content.pm.PackageManager
+import android.os.Build
 import me.amrbashir.hijriwidget.BuildConfig
 import me.amrbashir.hijriwidget.HijriDate
 import me.amrbashir.hijriwidget.PreferencesManager
 
 fun changeLauncherIcon(context: Context, prefsManager: PreferencesManager) {
-    val today = HijriDate.todayNumber(prefsManager)
+    val actions = buildList {
+        if (!BuildConfig.DEBUG) {
+            val cls = "me.amrbashir.hijriwidget.preference_activity.PreferenceActivity"
+            val mainActivity = ComponentName(context, cls)
+            add(mainActivity to PackageManager.COMPONENT_ENABLED_STATE_DISABLED)
```

Comment:
Nitpick: I dislike usage of `to` coming from different background of programming languages and I feel like it would be better to explicitly use the `Pair` constructor instead
```suggestion
            add(Pair(mainActivity, PackageManager.COMPONENT_ENABLED_STATE_DISABLED))
```